### ISSUE TITLE
Triangle doesn't render well in Mobile Safari

### ISSUE THREAD

**Issue by Wolfr** [lemonmade/bigfoot#42]
Screenshot from iOS 8.4 simulator:

http://cl.ly/image/3x2c2e1D1a2E/Screen%20Shot%202015-07-20%20at%2008.45.46.png

If I find a fix I will PR.

**Comment by Wolfr** [comment]
The same occured in normal Safari btw.

**Comment by Wolfr** [comment]
Look at it from a person finding bigfoot and just wanting it in their project (it's awesome btw).

I just use a very simple `compass watch` for this project, not even a grunt build system. Adding autoprefixer is unnecessary in this simple context.

So anyone coming to this project is supposed to find these files and combine them?

https://raw.githubusercontent.com/lemonmade/bigfoot/master/dist/bigfoot-bottom.css
https://raw.githubusercontent.com/lemonmade/bigfoot/master/dist/bigfoot-default.css
https://raw.githubusercontent.com/lemonmade/bigfoot/master/dist/bigfoot-number.css

In my eyes a dist folder is supposed to contain only the .css file (all 3 together and with comments), the source (scss) lives outside of the dist folder.

Ideally the dist folder contains a minified version of the CSS as well.

The SCSS can be used to generate the CSS and can have requirements (like autoprefixer); now you leave authors with 3 uncommented files that they have to combine themselves, after which they are left with uncommented code - if they don't use a similar build system.

**Comment by lemonmade** [comment]
I understand where you are coming from, but I think yours is a less common use case than you would think. For one, most people use one of those CSS files and make small adjustments — most aren't combining elements from different styles, and considering there are only two button styles and two popover styles, the fact that I have three out of the four possible combinations is, I think, reasonable.

There is no way to handle this that pleases everyone — if I leave them out, people using the Sass without a full build system get left in the code, and if I put them in, people get locked into potentially outdated, bloat-inducing prefixes, and I incur a bunch of overhead in maintaining what prefixes get added. I think the Sass + no build system is the least common use case, so I optimize the other way.

The `dist` folder being a mess and CSS files having no comments is a separate issue that I think does have merit. My rationale for the uncommented CSS was that a) the class names were easy to follow, and basically documented, b) people would be making small changes, not big ones, and c) I assume that people grabbing the CSS are unlikely to have a minification build step, so I wanted to err on the side of reducing bloat. That might not be the best rationale, but it's what I went with. The Sass in the `dist` folder is also weird — it's mostly there because I wanted to provide a combined Sass file for each of the three themes (including button + popover), but wanted those to be separate files (since they are shared between themes), so there really isn't a logical place to put those combined Sass where they won't be accidentally edited.

**Comment by Wolfr** [comment]
Hi @lemonmade , thanks for the detailed reasoning.

I think we were misunderstanding each other.

I think the issue is that I didn't know the difference between the 3 CSS files, it's not explained on the bigfoot website. All I wanted was just to get bigfoot on my website as fast as possible with default styling (the popover).

So I find this project, and copy the JS and CSS over and test if it works without reading too much docs, and I'm a bit confused on what exactly to copy. There is no description of this on the github overview or on the website (it's a shame I didn't go through it more before because the website is really beautiful :)) - I think that's pretty common behavior for most web people.

I read the source and it's clear - now I know the difference between number and bottom and default; I only need default. My fault.

**Comment by lemonmade** [comment]
I agree — the documentation in the repo could be much improved. I will tackle that at some point in the near future — this project started as a project for people who would just download a zip file from the website, but has evolved into people with more technical leanings downloading/ cloning from Github, and I never updated the documentation to match.

Sorry if I came off the wrong way; I appreciate the feedback!
